frontends/halo2-core/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "halo2-frontend-core"
+version = "0.1.0"
+edition = "2024"
+
+[dependencies]
+ff = { workspace = true }
+anyhow = "1" # TODO: Change this for a proper error type.
```

Comment:
Is this a TODO for a different PR?

Reply:
Yeah, I don't wanna deal with that rn since it would require going through every usage of anyhow and creating an error for that case